process/block/preprocess/scheduledTxsExecution.go

```diff
@@ -285,6 +305,96 @@ func (ste *scheduledTxsExecution) HaveScheduledTxs() bool {
 	return numScheduledTxs > 0
 }
 
+// GetScheduledRootHashForHeader gets scheduled root hash of the given header from storage
+func (ste *scheduledTxsExecution) GetScheduledRootHashForHeader(
+	headerHash []byte,
+) ([]byte, error) {
+	rootHash, _, err := ste.getScheduledRootHashAndSCRsForHeader(headerHash)
+	return rootHash, err
+}
+
+// RollBackToBlock rolls back the scheduled txs execution handler to the given header
+func (ste *scheduledTxsExecution) RollBackToBlock(headerHash []byte) error {
+	scheduledRootHash, mapScheduledSCRs, err := ste.getScheduledRootHashAndSCRsForHeader(headerHash)
+	ste.SetScheduledRootHashAndSCRs(scheduledRootHash, mapScheduledSCRs)
+
+	return err
```

Comment:
return nil here and if err != nil { return err } before L319 ?

Reply:
done